nextcord/__init__.py

```diff
@@ -13,7 +13,7 @@
 __author__ = 'tag-epic & Rapptz'
 __license__ = 'MIT'
 __copyright__ = 'Copyright 2015-present Rapptz'
-__version__ = '2.0.0a3'
+__version__ = '2.0.0a4'
```

Comment:
i assume you asked epic about version bumping?

Reply:
Don't bump the version

Reply:
So versionadded will be a3 then in docs?

Reply:
No it should just be 2.0.0, as the ``a`` means alpha and it's not a proper release